modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaUpgradeModelPermissionsCheck.java

```diff
@@ -199,11 +204,18 @@ private String _getModelPermissionsImplementation(
 	}
 
 	private boolean _isServiceContextMethodCall(
-		String methodCall, String fileContent, String variableName) {
+			String methodCall, String fileContent, String fileName,
+			String variableName)
```

Comment:
@TamyrisTorres, can you please sort alphabetically?

Reply:
@natocesarrego resolved